src/black/__init__.py

```diff
@@ -494,6 +494,7 @@ def main(
                 workers=workers,
             )
 
+    out()
     if verbose or not quiet:
         out(error_msg if report.return_code else "All done! ✨ 🍰 ✨")
```

Comment:
```suggestion
    if verbose or not quiet:
        out()
        out(error_msg if report.return_code else "All done! ✨ 🍰 ✨")
```
Sorry for being unclear, but we should only add the newline if we're gonna be emitting the summary anyway.

Reply:
Oh I apologize! That would be an oversight from me